src/integrationtests/resources/com/aws/greengrass/integrationtests/metrics/metricsConfig.yaml

```diff
@@ -0,0 +1,26 @@
+---
+services:
+  aws.greengrass.Nucleus:
+    configuration:
+      runWithDefault:
+        posixUser: nobody
+        windowsUser: integ-tester
+      logging:
+        level: "DEBUG"
+  aws.greengrass.clientdevices.Auth:
+    configuration:
+      deviceGroups:
+        formatVersion: "2021-03-05"
+        definitions:
+          devices:
+            selectionRule: "*"
+            policyName: "allowPublishMetrics"
+        policies:
+          allowPublishMetrics:
+            policyStatement1:
+              statementDescription: "publish metrics"
+              operations: "*"
+              resources: "*"
+      metrics:
+        disableMetrics: "false"
+        aggregatePeriod: "2"
```

Comment:
what is the unit of the aggregation period? The unit should be in the name: `aggregationPeriodSeconds`

Reply:
I'll update this in the config